docs/internal-documentation/distrib-testing/BuildEnvironmentPreparation.md

```diff
@@ -0,0 +1,78 @@
+# Build Environment Preparation
+
+## Overview
+
+The build environment is used for several purposes, but the purpose
+is not for the actual installation of Zonemaster. For the installation
+the normal installation instructions for users should be used. In most
+cases, if you create a build environment you usually do not install
+Zonemaster on that.
+
+* When testing the installation instructions for users you should avoid
+  to do that from a build environment.
+* For the use cases below, your should in most cases make sure that
+  you use the develop branch.
+* You should probably read this file from the develop branch, not
+  master branch to get the latest changes.
```

Comment:
It would be convenient to have a link here.

Reply:
I will fix all three comments in a new PR.